warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -1331,7 +1335,13 @@ public static void logDebug(List<String> output, String message) {
     }
     
     public static void logQuery(ASTJexlScript queryTree, String message) {
-        logDebug(PrintingVisitor.formattedQueryStringList(queryTree), message);
```

Comment:
@ivakegg Thoughts?  This would print the simple query string instead of the formatted query tree if the simple query string is over 1000000 characters (which is common for geo queries).

Reply:
I would like to improve the side of the formattedQueryString.    When we have over N children at a node, then print the first 5, ... N-10, last 5 instead of the entire set.

Reply:
or instead of printing ... (N-10), print ... (n-10 of X, Y, Z) where X, Y, Z are the unique set of fields found in the skipped children.